lib/queryBuilder/QueryBuilder.js

```diff
@@ -630,43 +630,63 @@ class QueryBuilder extends QueryBuilderBase {
   }
 
   map(...args) {
+    deprecate(
+      'map is deprecated. Use the modify method instead. applyModifier will be removed in objection 3.0'
```

Comment:
comment is wrong. Should be: map is deprecated. map will be removed in objection 3.0